certifikit-cli/build.gradle

```diff
@@ -36,15 +39,35 @@ dependencies {
 }
 
 application {
-  mainClass = 'app.cash.certifikit.cli.Main'
+  mainClassName('app.cash.certifikit.cli.Main')
 }
 
+def reflectionOutputFile = "${buildDir}/resources/main/META-INF/native-image/${project.group}/${project.name}/reflect-config.json"
+
 task(generateGraalReflectionConfig, dependsOn: 'classes', type: JavaExec) {
   main = 'picocli.codegen.aot.graalvm.ReflectionConfigGenerator'
   classpath = configurations.generateConfig + sourceSets.main.runtimeClasspath
-  def outputFile = "${buildDir}/resources/main/META-INF/native-image/${project.group}/${project.name}/reflect-config.json"
-  args = ["--output=$outputFile", 'app.cash.certifikit.cli.Main']
+  args = ["--output=$reflectionOutputFile", 'app.cash.certifikit.cli.Main']
 }
 assemble.dependsOn generateGraalReflectionConfig
 assemble.dependsOn installShadowDist
 
+graal {
+  mainClass("app.cash.certifikit.cli.Main")
+  outputName("cft")
+  graalVersion('20.1.0')
```

Comment:
You can reference this value in the `dependencies` block until https://github.com/palantir/gradle-graal/issues/377 is available.